app/src/main/java/cs2063/groceryshopper/TripActivity.kt

```diff
@@ -123,14 +128,38 @@ class TripActivity : AppCompatActivity() {
         }
     }
 
+    // TODO: BEN -> Figure out how to recall Database
```

Comment:
Here @bjacobsUNB

Reply:
Done, i made the DB and Trip Id global so you can access them from that function. This is fine because there will only ever be one trip open at a time so the global variables wont get overwritten